src/MetalPerformanceShaders/MPSKernel.cs

```diff
@@ -250,6 +250,54 @@ public MPSImageGaussianPyramid (IMTLDevice device, nuint kernelWidth, nuint kern
 			}
 		}
 	}
+
+	public partial class MPSImageLaplacianPyramid {
+		[DesignatedInitializer]
+		public MPSImageLaplacianPyramid (IMTLDevice device, nuint kernelWidth, nuint kernelHeight, float[] kernelWeights) : base (NSObjectFlag.Empty)
+		{
+			if (kernelWeights == null)
+				throw new ArgumentNullException (nameof (kernelWeights));
+			if (kernelWidth * kernelHeight != (nuint) kernelWeights.Length)
```

Comment:
Technically `kernelWidth * kernelHeight <= kernelWeights.Length` is good enough to be safe (and the exception message would be 'size must be _at least_ 'kernelHeight * kernelWidth')